ee/modules/600-flant-integration/images/flant-pricing/werf.inc.yaml

```diff
@@ -0,0 +1,75 @@
+{{- $shellOperatorVersion := "v1.3.1"}}
+{{- $jqVersion := "b6be13d5"}}
+---
+image: {{ .ModuleName }}/flant-pricing
+from: {{ .Images.BASE_ALT }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /shell-operator/shell-operator
+  to: /shell-operator
+  before: setup
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /shell-operator/shell_lib.sh
+  to: /shell_lib.sh
+  before: setup
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /shell-operator/frameworks
+  to: /frameworks
+  before: setup
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /jq/jq
+  to: /usr/local/bin/jq
```

Comment:
we have prebuilt common/shell-operator image

Reply:
done